src/ArticleCard.jsx

```diff
@@ -0,0 +1,10 @@
+function ArticleCard({articles}) {
+    return(
+        <ul className="article-card">
+            <img src={articles.article_img_url}/>
+            <h3>{articles.title}</h3>
```

Comment:
We should be using Headings in descending order - I can't find an H2 in your article, so realistically, this should be an H2 and styled accordingly